tests/utils.go

```diff
@@ -2810,6 +2811,8 @@ func NewConsoleExpecter(virtCli kubecli.KubevirtClient, vmi *v1.VirtualMachineIn
 	}()
 
 	opts = append(opts, expect.SendTimeout(timeout))
+	opts = append(opts, expect.Verbose(true))
```

Comment:
Thats cool. but consider that on fedora (for example test 3237)
you will see this output
```
==========================
Random Seed: 1592298479
Will run 1 of 561 specs

SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
------------------------------
[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system] VM Live Migration Starting a VirtualMachineInstance  with auto converge enabled 
  [test_id:3237]should complete a migration
  /root/go/src/kubevirt.io/kubevirt/tests/migration_test.go:483
STEP: Starting the VirtualMachineInstance
STEP: Starting a VirtualMachineInstance
STEP: Waiting until the VirtualMachineInstance starts
{"component":"tests","kind":"VirtualMachineInstance","level":"info","msg":"Event(v1.ObjectReference{Kind:\"VirtualMachineInstance\", Namespace:\"kubevirt-test-default\", Name:\"testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd\", UID:\"50be8208-0493-4e5d-8240-85f084c73c46\", APIVersion:\"kubevirt.io/v1alpha3\", ResourceVersion:\"11674\", FieldPath:\"\"}): type: 'Normal' reason: 'SuccessfulCreate' Created virtual machine pod virt-launcher-testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc42zt8wx","name":"testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd","namespace":"kubevirt-test-default","pos":"utils.go:370","timestamp":"2020-06-16T09:08:16.170218Z","uid":"50be8208-0493-4e5d-8240-85f084c73c46"}
{"component":"tests","kind":"VirtualMachineInstance","level":"info","msg":"Event(v1.ObjectReference{Kind:\"VirtualMachineInstance\", Namespace:\"kubevirt-test-default\", Name:\"testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd\", UID:\"50be8208-0493-4e5d-8240-85f084c73c46\", APIVersion:\"kubevirt.io/v1alpha3\", ResourceVersion:\"11844\", FieldPath:\"\"}): type: 'Normal' reason: 'Created' VirtualMachineInstance defined.","name":"testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd","namespace":"kubevirt-test-default","pos":"utils.go:370","timestamp":"2020-06-16T09:08:50.225172Z","uid":"50be8208-0493-4e5d-8240-85f084c73c46"}
{"component":"tests","kind":"VirtualMachineInstance","level":"info","msg":"Event(v1.ObjectReference{Kind:\"VirtualMachineInstance\", Namespace:\"kubevirt-test-default\", Name:\"testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd\", UID:\"50be8208-0493-4e5d-8240-85f084c73c46\", APIVersion:\"kubevirt.io/v1alpha3\", ResourceVersion:\"11875\", FieldPath:\"\"}): type: 'Normal' reason: 'Started' VirtualMachineInstance started.","name":"testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd","namespace":"kubevirt-test-default","pos":"utils.go:370","timestamp":"2020-06-16T09:08:50.248897Z","uid":"50be8208-0493-4e5d-8240-85f084c73c46"}
STEP: Checking that the VirtualMachineInstance console has expected output
Sent: "\n"
Match for RE: "login:" found: ["login:"] Buffer: [    0.000000] Linux version 5.6.6-300.fc32.x86_64 (mockbuild@bkernel03.phx2.fedoraproject.org) (gcc version 10.0.1 20200328 (Red Hat 10.0.1-0.11) (GCC)) #1 SMP Tue Apr 21 13:44:19 UTC 2020
[    0.000000] Command line: BOOT_IMAGE=(hd0,msdos1)/boot/vmlinuz-5.6.6-300.fc32.x86_64 root=UUID=d1b37ed4-3bbb-40b2-a6ba-f377f0c90217 ro no_timer_check net.ifnames=0 console=tty1 console=ttyS0,115200n8
[    0.000000] x86/fpu: x87 FPU will use FXSAVE
[    0.000000] BIOS-provided physical RAM map:
[    0.000000] BIOS-e820: [mem 0x0000000000000000-0x000000000009fbff] usable
[    0.000000] BIOS-e820: [mem 0x000000000009fc00-0x000000000009ffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000000f0000-0x00000000000fffff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000000100000-0x000000000f4dbfff] usable
[    0.000000] BIOS-e820: [mem 0x000000000f4dc000-0x000000000f4fffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000b0000000-0x00000000bfffffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000fed1c000-0x00000000fed1ffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000feffc000-0x00000000feffffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000fffc0000-0x00000000ffffffff] reserved
[    0.000000] NX (Execute Disable) protection: active
[    0.000000] SMBIOS 2.8 present.
[    0.000000] DMI: KubeVirt None/RHEL-AV, BIOS ?-20190727_073836-buildvm-ppc64le-16.ppc.fedoraproject.org-3.fc31 04/01/2014
[    0.000000] Hypervisor detected: KVM
[    0.000000] kvm-clock: Using msrs 4b564d01 and 4b564d00
[    0.000000] kvm-clock: cpu 0, msr 7601001, primary cpu clock
[    0.000000] kvm-clock: using sched offset of 4666956931 cycles
[    0.000009] clocksource: kvm-clock: mask: 0xffffffffffffffff max_cycles: 0x1cd42e4dffb, max_idle_ns: 881590591483 ns
[    0.000015] tsc: Detected 2400.084 MHz processor
[    0.003077] last_pfn = 0xf4dc max_arch_pfn = 0x400000000
[    0.003166] x86/PAT: Configuration [0-7]: WB  WC  UC- UC  WB  WP  UC- WT  
[    0.014611] found SMP MP-table at [mem 0x000f5cc0-0x000f5ccf]
[    0.019213] Using GB pages for direct mapping
[    0.019629] RAMDISK: [mem 0x0c626000-0x0de73fff]
[    0.019690] ACPI: Early table checksum verification disabled
[    0.019731] ACPI: RSDP 0x00000000000F5A50 000014 (v00 BOCHS )
[    0.019741] ACPI: RSDT 0x000000000F4E20A6 000030 (v01 BOCHS  BXPCRSDT 00000001 BXPC 00000001)
[    0.019763] ACPI: FACP 0x000000000F4E1EFE 0000F4 (v03 BOCHS  BXPCFACP 00000001 BXPC 00000001)
[    0.019775] ACPI: DSDT 0x000000000F4E0040 001EBE (v01 BOCHS  BXPCDSDT 00000001 BXPC 00000001)
[    0.019780] ACPI: FACS 0x000000000F4E0000 000040
[    0.019785] ACPI: APIC 0x000000000F4E1FF2 000078 (v01 BOCHS  BXPCAPIC 00000001 BXPC 00000001)
[    0.019790] ACPI: MCFG 0x000000000F4E206A 00003C (v01 BOCHS  BXPCMCFG 00000001 BXPC 00000001)
[    0.020342] No NUMA configuration found
[    0.020344] Faking a node at [mem 0x0000000000000000-0x000000000f4dbfff]
[    0.020371] NODE_DATA(0) allocated [mem 0x0f4b1000-0x0f4dbfff]
[    0.021637] Zone ranges:
[    0.021643]   DMA      [mem 0x0000000000001000-0x0000000000ffffff]
[    0.021645]   DMA32    [mem 0x0000000001000000-0x000000000f4dbfff]
[    0.021647]   Normal   empty
[    0.021648]   Device   empty
[    0.021649] Movable zone start for each node
[    0.021652] Early memory node ranges
[    0.021654]   node   0: [mem 0x0000000000001000-0x000000000009efff]
[    0.021655]   node   0: [mem 0x0000000000100000-0x000000000f4dbfff]
[    0.021691] Zeroed struct page in unavailable ranges: 2950 pages
[    0.021693] Initmem setup node 0 [mem 0x0000000000001000-0x000000000f4dbfff]
[    0.024240] ACPI: PM-Timer IO Port: 0x608
[    0.024272] ACPI: LAPIC_NMI (acpi_id[0xff] dfl dfl lint[0x1])
[    0.024342] IOAPIC[0]: apic_id 0, version 17, address 0xfec00000, GSI 0-23
[    0.024347] ACPI: INT_SRC_OVR (bus 0 bus_irq 0 global_irq 2 dfl dfl)
[    0.024350] ACPI: INT_SRC_OVR (bus 0 bus_irq 5 global_irq 5 high level)
[    0.024351] ACPI: INT_SRC_OVR (bus 0 bus_irq 9 global_irq 9 high level)
[    0.024358] ACPI: INT_SRC_OVR (bus 0 bus_irq 10 global_irq 10 high level)
[    0.024359] ACPI: INT_SRC_OVR (bus 0 bus_irq 11 global_irq 11 high level)
[    0.024371] Using ACPI (MADT) for SMP configuration information
[    0.024387] smpboot: Allowing 1 CPUs, 0 hotplug CPUs
[    0.024447] PM: hibernation: Registered nosave memory: [mem 0x00000000-0x00000fff]
[    0.024449] PM: hibernation: Registered nosave memory: [mem 0x0009f000-0x0009ffff]
[    0.024450] PM: hibernation: Registered nosave memory: [mem 0x000a0000-0x000effff]
[    0.024451] PM: hibernation: Registered nosave memory: [mem 0x000f0000-0x000fffff]
[    0.024453] [mem 0x0f500000-0xafffffff] available for PCI devices
[    0.024454] Booting paravirtualized kernel on KVM
[    0.024470] clocksource: refined-jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 1910969940391419 ns
[    0.030757] setup_percpu: NR_CPUS:8192 nr_cpumask_bits:1 nr_cpu_ids:1 nr_node_ids:1
[    0.031242] percpu: Embedded 55 pages/cpu s188416 r8192 d28672 u2097152
[    0.031300] KVM setup async PF for cpu 0
[    0.031307] kvm-stealtime: cpu 0, msr ee2d040
[    0.031327] Built 1 zonelists, mobility grouping on.  Total pages: 61585
[    0.031328] Policy zone: DMA32
[    0.031330] Kernel command line: BOOT_IMAGE=(hd0,msdos1)/boot/vmlinuz-5.6.6-300.fc32.x86_64 root=UUID=d1b37ed4-3bbb-40b2-a6ba-f377f0c90217 ro no_timer_check net.ifnames=0 console=tty1 console=ttyS0,115200n8
[    0.031922] Dentry cache hash table entries: 32768 (order: 6, 262144 bytes, linear)
[    0.032083] Inode-cache hash table entries: 16384 (order: 5, 131072 bytes, linear)
[    0.032181] mem auto-init: stack:off, heap alloc:off, heap free:off
[    0.032930] Memory: 179436K/250344K available (14339K kernel code, 2400K rwdata, 4868K rodata, 2452K init, 6136K bss, 70908K reserved, 0K cma-reserved)
[    0.032984] random: get_random_u64 called from __kmem_cache_create+0x3e/0x620 with crng_init=0
[    0.033208] SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
[    0.033244] Kernel/User page tables isolation: enabled
[    0.033280] ftrace: allocating 40698 entries in 159 pages
[    0.056933] ftrace: allocated 159 pages with 6 groups
[    0.057084] rcu: Hierarchical RCU implementation.
[    0.057086] rcu: 	RCU restricting CPUs from NR_CPUS=8192 to nr_cpu_ids=1.
[    0.057088] 	Tasks RCU enabled.
[    0.057089] rcu: RCU calculated value of scheduler-enlistment delay is 100 jiffies.
[    0.057090] rcu: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=1
[    0.061296] NR_IRQS: 524544, nr_irqs: 256, preallocated irqs: 16
[    0.090929] Console: colour VGA+ 80x25
[    0.157765] printk: console [tty1] enabled
[    0.338620] printk: console [ttyS0] enabled
[    0.340746] ACPI: Core revision 20200110
[    0.342829] APIC: Switch to symmetric I/O mode setup
[    0.345697] x2apic enabled
[    0.347662] Switched APIC routing to physical x2apic.
[    0.352307] clocksource: tsc-early: mask: 0xffffffffffffffff max_cycles: 0x229886f1f14, max_idle_ns: 440795306808 ns
[    0.357106] Calibrating delay loop (skipped) preset value.. 4800.16 BogoMIPS (lpj=2400084)
[    0.358102] pid_max: default: 32768 minimum: 301
[    0.358102] LSM: Security Framework initializing
[    0.358102] Yama: becoming mindful.
[    0.358102] SELinux:  Initializing.
[    0.358102] *** VALIDATE selinux ***
[    0.358102] Mount-cache hash table entries: 512 (order: 0, 4096 bytes, linear)
[    0.358102] Mountpoint-cache hash table entries: 512 (order: 0, 4096 bytes, linear)
[    0.358102] *** VALIDATE tmpfs ***
[    0.358102] *** VALIDATE proc ***
[    0.358102] *** VALIDATE cgroup ***
[    0.358102] *** VALIDATE cgroup2 ***
[    0.358102] x86/cpu: User Mode Instruction Prevention (UMIP) activated
[    0.358102] Last level iTLB entries: 4KB 0, 2MB 0, 4MB 0
[    0.358102] Last level dTLB entries: 4KB 0, 2MB 0, 4MB 0, 1GB 0
[    0.358102] Spectre V1 : Mitigation: usercopy/swapgs barriers and __user pointer sanitization
[    0.358102] Spectre V2 : Mitigation: Full generic retpoline
[    0.358102] Spectre V2 : Spectre v2 / SpectreRSB mitigation: Filling RSB on context switch
[    0.358102] Speculative Store Bypass: Vulnerable
[    0.358102] MDS: Vulnerable: Clear CPU buffers attempted, no microcode
[    0.358102] Freeing SMP alternatives memory: 36K
[    0.358102] smpboot: CPU0: Intel Westmere E56xx/L56xx/X56xx (Nehalem-C) (family: 0x6, model: 0x2c, stepping: 0x1)
[    0.358282] Performance Events: unsupported p6 CPU model 44 no PMU driver, software events only.
[    0.359183] rcu: Hierarchical SRCU implementation.
[    0.360817] NMI watchdog: Perf NMI watchdog permanently disabled
[    0.361178] smp: Bringing up secondary CPUs ...
[    0.362114] smp: Brought up 1 node, 1 CPU
[    0.363110] smpboot: Max logical packages: 1
[    0.364111] smpboot: Total of 1 processors activated (4800.16 BogoMIPS)
[    0.365323] devtmpfs: initialized
[    0.366165] x86/mm: Memory block size: 128MB
[    0.367546] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 1911260446275000 ns
[    0.368117] futex hash table entries: 256 (order: 2, 16384 bytes, linear)
[    0.369178] pinctrl core: initialized pinctrl subsystem
[    0.370323] PM: RTC time: 09:08:54, date: 2020-06-16
[    0.371112] thermal_sys: Registered thermal governor 'fair_share'
[    0.371113] thermal_sys: Registered thermal governor 'bang_bang'
[    0.372110] thermal_sys: Registered thermal governor 'step_wise'
[    0.373116] thermal_sys: Registered thermal governor 'user_space'
[    0.374311] NET: Registered protocol family 16
[    0.376259] audit: initializing netlink subsys (disabled)
[    0.377268] cpuidle: using governor menu
[    0.378236] ACPI: bus type PCI registered
[    0.379110] acpiphp: ACPI Hot Plug PCI Controller Driver version: 0.5
[    0.380327] PCI: MMCONFIG for domain 0000 [bus 00-ff] at [mem 0xb0000000-0xbfffffff] (base 0xb0000000)
[    0.381113] PCI: MMCONFIG at [mem 0xb0000000-0xbfffffff] reserved in E820
[    0.382124] PCI: Using configuration type 1 for base access
[    0.385278] audit: type=2000 audit(1592298534.940:1): state=initialized audit_enabled=0 res=1
[    0.386257] HugeTLB registered 1.00 GiB page size, pre-allocated 0 pages
[    0.387111] HugeTLB registered 2.00 MiB page size, pre-allocated 0 pages
[    0.617262] cryptd: max_cpu_qlen set to 1000
[    0.620327] alg: No test for 842 (842-generic)
[    0.621247] alg: No test for 842 (842-scomp)
[    0.626995] ACPI: Added _OSI(Module Device)
[    0.627121] ACPI: Added _OSI(Processor Device)
[    0.628112] ACPI: Added _OSI(3.0 _SCP Extensions)
[    0.629111] ACPI: Added _OSI(Processor Aggregator Device)
[    0.630126] ACPI: Added _OSI(Linux-Dell-Video)
[    0.631167] ACPI: Added _OSI(Linux-Lenovo-NV-HDMI-Audio)
[    0.632120] ACPI: Added _OSI(Linux-HPI-Hybrid-Graphics)
[    0.634479] ACPI: 1 ACPI AML tables successfully acquired and loaded
[    0.636311] ACPI: Interpreter enabled
[    0.637130] ACPI: (supports S0 S5)
[    0.638110] ACPI: Using IOAPIC for interrupt routing
[    0.639143] PCI: Using host bridge windows from ACPI; if necessary, use "pci=nocrs" and report a bug
[    0.640236] ACPI: Enabled 1 GPEs in block 00 to 3F
[    0.643740] ACPI: PCI Root Bridge [PCI0] (domain 0000 [bus 00-ff])
[    0.644117] acpi PNP0A08:00: _OSC: OS supports [ExtendedConfig ASPM ClockPM Segments MSI HPX-Type3]
[    0.645210] acpi PNP0A08:00: _OSC: platform does not support [LTR]
[    0.646197] acpi PNP0A08:00: _OSC: OS now controls [PCIeHotplug SHPCHotplug PME AER PCIeCapability]
[    0.647294] PCI host bridge to bus 0000:00
[    0.648112] pci_bus 0000:00: root bus resource [io  0x0000-0x0cf7 window]
[    0.649111] pci_bus 0000:00: root bus resource [io  0x0d00-0xffff window]
[    0.650111] pci_bus 0000:00: root bus resource [mem 0x000a0000-0x000bffff window]
[    0.651111] pci_bus 0000:00: root bus resource [mem 0x0f500000-0xafffffff window]
[    0.652112] pci_bus 0000:00: root bus resource [mem 0xc0000000-0xfebfffff window]
[    0.653112] pci_bus 0000:00: root bus resource [mem 0x100000000-0x8ffffffff window]
[    0.654112] pci_bus 0000:00: root bus resource [bus 00-ff]
[    0.655182] pci 0000:00:00.0: [8086:29c0] type 00 class 0x060000
[    0.657111] pci 0000:00:01.0: [1234:1111] type 00 class 0x030000
[    0.660150] pci 0000:00:01.0: reg 0x10: [mem 0xfc000000-0xfcffffff pref]
[    0.665155] pci 0000:00:01.0: reg 0x18: [mem 0xfea10000-0xfea10fff]
[    0.672167] pci 0000:00:01.0: reg 0x30: [mem 0xfea00000-0xfea0ffff pref]
[    0.673749] pci 0000:00:02.0: [1b36:000c] type 01 class 0x060400
[    0.676112] pci 0000:00:02.0: reg 0x10: [mem 0xfea11000-0xfea11fff]
[    0.681280] pci 0000:00:02.1: [1b36:000c] type 01 class 0x060400
[    0.684080] pci 0000:00:02.1: reg 0x10: [mem 0xfea12000-0xfea12fff]
[    0.688675] pci 0000:00:02.2: [1b36:000c] type 01 class 0x060400
[    0.690876] pci 0000:00:02.2: reg 0x10: [mem 0xfea13000-0xfea13fff]
[    0.695721] pci 0000:00:02.3: [1b36:000c] type 01 class 0x060400
[    0.697905] pci 0000:00:02.3: reg 0x10: [mem 0xfea14000-0xfea14fff]
[    0.702038] pci 0000:00:02.4: [1b36:000c] type 01 class 0x060400
[    0.704112] pci 0000:00:02.4: reg 0x10: [mem 0xfea15000-0xfea15fff]
[    0.709014] pci 0000:00:02.5: [1b36:000c] type 01 class 0x060400
[    0.711112] pci 0000:00:02.5: reg 0x10: [mem 0xfea16000-0xfea16fff]
[    0.716388] pci 0000:00:1f.0: [8086:2918] type 00 class 0x060100
[    0.718008] pci 0000:00:1f.0: quirk: [io  0x0600-0x067f] claimed by ICH6 ACPI/GPIO/TCO
[    0.718505] pci 0000:00:1f.2: [8086:2922] type 00 class 0x010601
[    0.724724] pci 0000:00:1f.2: reg 0x20: [io  0xc040-0xc05f]
[    0.726131] pci 0000:00:1f.2: reg 0x24: [mem 0xfea17000-0xfea17fff]
[    0.729437] pci 0000:00:1f.3: [8086:2930] type 00 class 0x0c0500
[    0.733427] pci 0000:00:1f.3: reg 0x20: [io  0x0700-0x073f]
[    0.737364] pci 0000:01:00.0: [1af4:1041] type 00 class 0x020000
[    0.740112] pci 0000:01:00.0: reg 0x14: [mem 0xfe840000-0xfe840fff]
[    0.746113] pci 0000:01:00.0: reg 0x20: [mem 0xfda00000-0xfda03fff 64bit pref]
[    0.748112] pci 0000:01:00.0: reg 0x30: [mem 0xfe800000-0xfe83ffff pref]
[    0.750832] pci 0000:00:02.0: PCI bridge to [bus 01]
[    0.751167] pci 0000:00:02.0:   bridge window [mem 0xfe800000-0xfe9fffff]
[    0.752166] pci 0000:00:02.0:   bridge window [mem 0xfda00000-0xfdbfffff 64bit pref]
[    0.754336] pci 0000:02:00.0: [1af4:1043] type 00 class 0x078000
[    0.757757] pci 0000:02:00.0: reg 0x14: [mem 0xfe600000-0xfe600fff]
[    0.761113] pci 0000:02:00.0: reg 0x20: [mem 0xfd800000-0xfd803fff 64bit pref]
[    0.764945] pci 0000:00:02.1: PCI bridge to [bus 02]
[    0.765166] pci 0000:00:02.1:   bridge window [mem 0xfe600000-0xfe7fffff]
[    0.766166] pci 0000:00:02.1:   bridge window [mem 0xfd800000-0xfd9fffff 64bit pref]
[    0.768375] pci 0000:03:00.0: [1af4:1042] type 00 class 0x010000
[    0.771112] pci 0000:03:00.0: reg 0x14: [mem 0xfe400000-0xfe400fff]
[    0.775112] pci 0000:03:00.0: reg 0x20: [mem 0xfd600000-0xfd603fff 64bit pref]
[    0.778765] pci 0000:00:02.2: PCI bridge to [bus 03]
[    0.779168] pci 0000:00:02.2:   bridge window [mem 0xfe400000-0xfe5fffff]
[    0.780167] pci 0000:00:02.2:   bridge window [mem 0xfd600000-0xfd7fffff 64bit pref]
[    0.782370] pci 0000:04:00.0: [1af4:1042] type 00 class 0x010000
[    0.785112] pci 0000:04:00.0: reg 0x14: [mem 0xfe200000-0xfe200fff]
[    0.789112] pci 0000:04:00.0: reg 0x20: [mem 0xfd400000-0xfd403fff 64bit pref]
[    0.792770] pci 0000:00:02.3: PCI bridge to [bus 04]
[    0.793166] pci 0000:00:02.3:   bridge window [mem 0xfe200000-0xfe3fffff]
[    0.794167] pci 0000:00:02.3:   bridge window [mem 0xfd400000-0xfd5fffff 64bit pref]
[    0.796400] pci 0000:05:00.0: [1af4:1044] type 00 class 0x00ff00
[    0.800510] pci 0000:05:00.0: reg 0x20: [mem 0xfd200000-0xfd203fff 64bit pref]
[    0.803432] pci 0000:00:02.4: PCI bridge to [bus 05]
[    0.804166] pci 0000:00:02.4:   bridge window [mem 0xfe000000-0xfe1fffff]
[    0.805166] pci 0000:00:02.4:   bridge window [mem 0xfd200000-0xfd3fffff 64bit pref]
[    0.807169] pci 0000:00:02.5: PCI bridge to [bus 06]
[    0.808167] pci 0000:00:02.5:   bridge window [mem 0xfde00000-0xfdffffff]
[    0.809167] pci 0000:00:02.5:   bridge window [mem 0xfd000000-0xfd1fffff 64bit pref]
[    0.815171] ACPI: PCI Interrupt Link [LNKA] (IRQs 5 *10 11)
[    0.816341] ACPI: PCI Interrupt Link [LNKB] (IRQs 5 *10 11)
[    0.817304] ACPI: PCI Interrupt Link [LNKC] (IRQs 5 10 *11)
[    0.818299] ACPI: PCI Interrupt Link [LNKD] (IRQs 5 10 *11)
[    0.819254] ACPI: PCI Interrupt Link [LNKE] (IRQs 5 *10 11)
[    0.820250] ACPI: PCI Interrupt Link [LNKF] (IRQs 5 *10 11)
[    0.821249] ACPI: PCI Interrupt Link [LNKG] (IRQs 5 10 *11)
[    0.822268] ACPI: PCI Interrupt Link [LNKH] (IRQs 5 10 *11)
[    0.823175] ACPI: PCI Interrupt Link [GSIA] (IRQs *16)
[    0.824127] ACPI: PCI Interrupt Link [GSIB] (IRQs *17)
[    0.825127] ACPI: PCI Interrupt Link [GSIC] (IRQs *18)
[    0.826126] ACPI: PCI Interrupt Link [GSID] (IRQs *19)
[    0.827126] ACPI: PCI Interrupt Link [GSIE] (IRQs *20)
[    0.828127] ACPI: PCI Interrupt Link [GSIF] (IRQs *21)
[    0.829128] ACPI: PCI Interrupt Link [GSIG] (IRQs *22)
[    0.830142] ACPI: PCI Interrupt Link [GSIH] (IRQs *23)
[    0.831719] iommu: Default domain type: Translated 
[    0.832249] pci 0000:00:01.0: vgaarb: setting as boot VGA device
[    0.833102] pci 0000:00:01.0: vgaarb: VGA device added: decodes=io+mem,owns=io+mem,locks=none
[    0.833113] pci 0000:00:01.0: vgaarb: bridge control possible
[    0.834114] vgaarb: loaded
[    0.835381] SCSI subsystem initialized
[    0.836224] ACPI: bus type USB registered
[    0.837148] usbcore: registered new interface driver usbfs
[    0.838135] usbcore: registered new interface driver hub
[    0.839130] usbcore: registered new device driver usb
[    0.840160] pps_core: LinuxPPS API ver. 1 registered
[    0.841109] pps_core: Software ver. 5.3.6 - Copyright 2005-2007 Rodolfo Giometti <giometti@linux.it>
[    0.842157] PTP clock support registered
[    0.843230] EDAC MC: Ver: 3.0.0
[    0.844837] PCI: Using ACPI for IRQ routing
[    0.947451] NetLabel: Initializing
[    0.948109] NetLabel:  domain hash size = 128
[    0.949109] NetLabel:  protocols = UNLABELED CIPSOv4 CALIPSO
[    0.950151] NetLabel:  unlabeled traffic allowed by default
[    0.951404] clocksource: Switched to clocksource kvm-clock
[    0.973585] *** VALIDATE bpf ***
[    0.975038] VFS: Disk quotas dquot_6.6.0
[    0.976587] VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
[    0.978981] *** VALIDATE ramfs ***
[    0.980357] *** VALIDATE hugetlbfs ***
[    0.981965] pnp: PnP ACPI init
[    0.984009] pnp: PnP ACPI: found 4 devices
[    0.992187] clocksource: acpi_pm: mask: 0xffffff max_cycles: 0xffffff, max_idle_ns: 2085701024 ns
[    0.995535] pci 0000:00:02.0: bridge window [io  0x1000-0x0fff] to [bus 01] add_size 1000
[    0.999069] pci 0000:00:02.1: bridge window [io  0x1000-0x0fff] to [bus 02] add_size 1000
[    1.002223] pci 0000:00:02.2: bridge window [io  0x1000-0x0fff] to [bus 03] add_size 1000
[    1.005931] pci 0000:00:02.3: bridge window [io  0x1000-0x0fff] to [bus 04] add_size 1000
[    1.009101] pci 0000:00:02.4: bridge window [io  0x1000-0x0fff] to [bus 05] add_size 1000
[    1.012094] pci 0000:00:02.5: bridge window [io  0x1000-0x0fff] to [bus 06] add_size 1000
[    1.015226] pci 0000:00:02.0: BAR 13: assigned [io  0x1000-0x1fff]
[    1.017504] pci 0000:00:02.1: BAR 13: assigned [io  0x2000-0x2fff]
[    1.019685] pci 0000:00:02.2: BAR 13: assigned [io  0x3000-0x3fff]
[    1.022214] pci 0000:00:02.3: BAR 13: assigned [io  0x4000-0x4fff]
[    1.024848] pci 0000:00:02.4: BAR 13: assigned [io  0x5000-0x5fff]
[    1.027483] pci 0000:00:02.5: BAR 13: assigned [io  0x6000-0x6fff]
[    1.029983] pci 0000:00:02.0: PCI bridge to [bus 01]
[    1.032228] pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
[    1.036228] pci 0000:00:02.0:   bridge window [mem 0xfe800000-0xfe9fffff]
[    1.039494] pci 0000:00:02.0:   bridge window [mem 0xfda00000-0xfdbfffff 64bit pref]
[    1.044666] pci 0000:00:02.1: PCI bridge to [bus 02]
[    1.046745] pci 0000:00:02.1:   bridge window [io  0x2000-0x2fff]
[    1.050086] pci 0000:00:02.1:   bridge window [mem 0xfe600000-0xfe7fffff]
[    1.053417] pci 0000:00:02.1:   bridge window [mem 0xfd800000-0xfd9fffff 64bit pref]
[    1.058694] pci 0000:00:02.2: PCI bridge to [bus 03]
[    1.060645] pci 0000:00:02.2:   bridge window [io  0x3000-0x3fff]
[    1.064760] pci 0000:00:02.2:   bridge window [mem 0xfe400000-0xfe5fffff]
[    1.068189] pci 0000:00:02.2:   bridge window [mem 0xfd600000-0xfd7fffff 64bit pref]
[    1.072152] pci 0000:00:02.3: PCI bridge to [bus 04]
[    1.073817] pci 0000:00:02.3:   bridge window [io  0x4000-0x4fff]
[    1.077347] pci 0000:00:02.3:   bridge window [mem 0xfe200000-0xfe3fffff]
[    1.080246] pci 0000:00:02.3:   bridge window [mem 0xfd400000-0xfd5fffff 64bit pref]
[    1.084293] pci 0000:00:02.4: PCI bridge to [bus 05]
[    1.085948] pci 0000:00:02.4:   bridge window [io  0x5000-0x5fff]
[    1.089500] pci 0000:00:02.4:   bridge window [mem 0xfe000000-0xfe1fffff]
[    1.092363] pci 0000:00:02.4:   bridge window [mem 0xfd200000-0xfd3fffff 64bit pref]
[    1.096381] pci 0000:00:02.5: PCI bridge to [bus 06]
[    1.098488] pci 0000:00:02.5:   bridge window [io  0x6000-0x6fff]
[    1.101559] pci 0000:00:02.5:   bridge window [mem 0xfde00000-0xfdffffff]
[    1.104427] pci 0000:00:02.5:   bridge window [mem 0xfd000000-0xfd1fffff 64bit pref]
[    1.108762] pci_bus 0000:00: resource 4 [io  0x0000-0x0cf7 window]
[    1.110722] pci_bus 0000:00: resource 5 [io  0x0d00-0xffff window]
[    1.112678] pci_bus 0000:00: resource 6 [mem 0x000a0000-0x000bffff window]
[    1.118267] pci_bus 0000:00: resource 7 [mem 0x0f500000-0xafffffff window]
[    1.121114] pci_bus 0000:00: resource 8 [mem 0xc0000000-0xfebfffff window]
[    1.123353] pci_bus 0000:00: resource 9 [mem 0x100000000-0x8ffffffff window]
[    1.126085] pci_bus 0000:01: resource 0 [io  0x1000-0x1fff]
[    1.127921] pci_bus 0000:01: resource 1 [mem 0xfe800000-0xfe9fffff]
[    1.129950] pci_bus 0000:01: resource 2 [mem 0xfda00000-0xfdbfffff 64bit pref]
[    1.132454] pci_bus 0000:02: resource 0 [io  0x2000-0x2fff]
[    1.134248] pci_bus 0000:02: resource 1 [mem 0xfe600000-0xfe7fffff]
[    1.136221] pci_bus 0000:02: resource 2 [mem 0xfd800000-0xfd9fffff 64bit pref]
[    1.138643] pci_bus 0000:03: resource 0 [io  0x3000-0x3fff]
[    1.140438] pci_bus 0000:03: resource 1 [mem 0xfe400000-0xfe5fffff]
[    1.142406] pci_bus 0000:03: resource 2 [mem 0xfd600000-0xfd7fffff 64bit pref]
[    1.144798] pci_bus 0000:04: resource 0 [io  0x4000-0x4fff]
[    1.146635] pci_bus 0000:04: resource 1 [mem 0xfe200000-0xfe3fffff]
[    1.148636] pci_bus 0000:04: resource 2 [mem 0xfd400000-0xfd5fffff 64bit pref]
[    1.151736] pci_bus 0000:05: resource 0 [io  0x5000-0x5fff]
[    1.154176] pci_bus 0000:05: resource 1 [mem 0xfe000000-0xfe1fffff]
[    1.156843] pci_bus 0000:05: resource 2 [mem 0xfd200000-0xfd3fffff 64bit pref]
[    1.160246] pci_bus 0000:06: resource 0 [io  0x6000-0x6fff]
[    1.162298] pci_bus 0000:06: resource 1 [mem 0xfde00000-0xfdffffff]
[    1.164532] pci_bus 0000:06: resource 2 [mem 0xfd000000-0xfd1fffff 64bit pref]
[    1.167315] NET: Registered protocol family 2
[    1.169356] tcp_listen_portaddr_hash hash table entries: 256 (order: 0, 4096 bytes, linear)
[    1.172512] TCP established hash table entries: 2048 (order: 2, 16384 bytes, linear)
[    1.175649] TCP bind hash table entries: 2048 (order: 3, 32768 bytes, linear)
[    1.178625] TCP: Hash tables configured (established 2048 bind 2048)
[    1.181283] UDP hash table entries: 256 (order: 1, 8192 bytes, linear)
[    1.183876] UDP-Lite hash table entries: 256 (order: 1, 8192 bytes, linear)
[    1.186601] NET: Registered protocol family 1
[    1.188617] NET: Registered protocol family 44
[    1.190535] pci 0000:00:01.0: Video device with shadowed ROM at [mem 0x000c0000-0x000dffff]
[    1.194055] PCI: CLS 0 bytes, default 64
[    1.195864] Trying to unpack rootfs image as initramfs...
[    1.620222] Freeing initrd memory: 24888K
[    1.622088] clocksource: tsc: mask: 0xffffffffffffffff max_cycles: 0x229886f1f14, max_idle_ns: 440795306808 ns
[    1.626584] Initialise system trusted keyrings
[    1.628396] Key type blacklist registered
[    1.630035] workingset: timestamp_bits=36 max_order=16 bucket_order=0
[    1.634332] zbud: loaded
[    1.636285] Platform Keyring initialized
[    1.646188] NET: Registered protocol family 38
[    1.648195] Key type asymmetric registered
[    1.650259] Asymmetric key parser 'x509' registered
[    1.652576] Block layer SCSI generic (bsg) driver version 0.4 loaded (major 246)
[    1.656301] io scheduler mq-deadline registered
[    1.658423] io scheduler kyber registered
[    1.660485] io scheduler bfq registered
[    1.662234] atomic64_test: passed for x86-64 platform with CX8 and with SSE
[    1.665869] PCI Interrupt Link [GSIG] enabled at IRQ 22
[    1.670961] pcieport 0000:00:02.0: PME: Signaling with IRQ 24
[    1.673901] pcieport 0000:00:02.0: AER: enabled with IRQ 24
[    1.676565] pcieport 0000:00:02.0: pciehp: Slot #0 AttnBtn+ PwrCtrl+ MRL- AttnInd+ PwrInd+ HotPlug+ Surprise+ Interlock+ NoCompl- LLActRep+
[    1.685883] pcieport 0000:00:02.1: PME: Signaling with IRQ 25
[    1.688063] pcieport 0000:00:02.1: AER: enabled with IRQ 25
[    1.689998] pcieport 0000:00:02.1: pciehp: Slot #0 AttnBtn+ PwrCtrl+ MRL- AttnInd+ PwrInd+ HotPlug+ Surprise+ Interlock+ NoCompl- LLActRep+
[    1.697236] pcieport 0000:00:02.2: PME: Signaling with IRQ 26
[    1.699385] pcieport 0000:00:02.2: AER: enabled with IRQ 26
[    1.701303] pcieport 0000:00:02.2: pciehp: Slot #0 AttnBtn+ PwrCtrl+ MRL- AttnInd+ PwrInd+ HotPlug+ Surprise+ Interlock+ NoCompl- LLActRep+
[    1.708472] pcieport 0000:00:02.3: PME: Signaling with IRQ 27
[    1.710602] pcieport 0000:00:02.3: AER: enabled with IRQ 27
[    1.712558] pcieport 0000:00:02.3: pciehp: Slot #0 AttnBtn+ PwrCtrl+ MRL- AttnInd+ PwrInd+ HotPlug+ Surprise+ Interlock+ NoCompl- LLActRep+
[    1.719226] pcieport 0000:00:02.4: PME: Signaling with IRQ 28
[    1.721366] pcieport 0000:00:02.4: AER: enabled with IRQ 28
[    1.723661] pcieport 0000:00:02.4: pciehp: Slot #0 AttnBtn+ PwrCtrl+ MRL- AttnInd+ PwrInd+ HotPlug+ Surprise+ Interlock+ NoCompl- LLActRep+
[    1.730320] pcieport 0000:00:02.5: PME: Signaling with IRQ 29
[    1.732437] pcieport 0000:00:02.5: AER: enabled with IRQ 29
[    1.734357] pcieport 0000:00:02.5: pciehp: Slot #0 AttnBtn+ PwrCtrl+ MRL- AttnInd+ PwrInd+ HotPlug+ Surprise+ Interlock+ NoCompl- LLActRep+
[    1.738441] pcieport 0000:00:02.5: pciehp: Slot(0-5): Link Up
[    1.741094] shpchp: Standard Hot Plug PCI Controller Driver version: 0.4
[    1.743328] input: Power Button as /devices/LNXSYSTM:00/LNXPWRBN:00/input/input0
[    1.745875] ACPI: Power Button [PWRF]
[    1.757825] Serial: 8250/16550 driver, 32 ports, IRQ sharing enabled
[    1.760149] 00:03: ttyS0 at I/O 0x3f8 (irq = 4, base_baud = 115200) is a 16550A
[    1.764876] Non-volatile memory driver v1.3
[    1.766903] Linux agpgart interface v0.103
[    1.769072] PCI Interrupt Link [GSIA] enabled at IRQ 16
[    1.772008] ahci 0000:00:1f.2: AHCI 0001.0000 32 slots 6 ports 1.5 Gbps 0x3f impl SATA mode
[    1.774711] ahci 0000:00:1f.2: flags: 64bit ncq only 
[    1.777743] random: fast init done
[    1.779057] scsi host0: ahci
[    1.780287] random: crng init done
[    1.781947] scsi host1: ahci
[    1.783303] scsi host2: ahci
[    1.784522] scsi host3: ahci
[    1.785744] scsi host4: ahci
[    1.786954] scsi host5: ahci
[    1.788169] ata1: SATA max UDMA/133 abar m4096@0xfea17000 port 0xfea17100 irq 30
[    1.790614] ata2: SATA max UDMA/133 abar m4096@0xfea17000 port 0xfea17180 irq 30
[    1.793082] ata3: SATA max UDMA/133 abar m4096@0xfea17000 port 0xfea17200 irq 30
[    1.795538] ata4: SATA max UDMA/133 abar m4096@0xfea17000 port 0xfea17280 irq 30
[    1.797982] ata5: SATA max UDMA/133 abar m4096@0xfea17000 port 0xfea17300 irq 30
[    1.800444] ata6: SATA max UDMA/133 abar m4096@0xfea17000 port 0xfea17380 irq 30
[    1.803604] libphy: Fixed MDIO Bus: probed
[    1.805276] ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
[    1.807531] ehci-pci: EHCI PCI platform driver
[    1.809337] ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
[    1.811799] ohci-pci: OHCI PCI platform driver
[    1.813567] uhci_hcd: USB Universal Host Controller Interface driver
[    1.816080] usbcore: registered new interface driver usbserial_generic
[    1.818529] usbserial: USB Serial support registered for generic
[    1.820716] i8042: PNP: PS/2 Controller [PNP0303:KBD,PNP0f13:MOU] at 0x60,0x64 irq 1,12
[    1.825096] serio: i8042 KBD port at 0x60,0x64 irq 1
[    1.827190] serio: i8042 AUX port at 0x60,0x64 irq 12
[    1.829245] mousedev: PS/2 mouse device common for all mice
[    1.831738] input: AT Translated Set 2 keyboard as /devices/platform/i8042/serio0/input/input1
[    1.835283] rtc_cmos 00:00: RTC can wake from S4
[    1.838163] input: VirtualPS/2 VMware VMMouse as /devices/platform/i8042/serio1/input/input4
[    1.841185] rtc_cmos 00:00: registered as rtc0
[    1.842846] rtc_cmos 00:00: alarms up to one day, y3k, 114 bytes nvram
[    1.845271] device-mapper: uevent: version 1.0.3
[    1.847393] input: VirtualPS/2 VMware VMMouse as /devices/platform/i8042/serio1/input/input3
[    1.850582] device-mapper: ioctl: 4.42.0-ioctl (2020-02-27) initialised: dm-devel@redhat.com
[    1.853716] intel_pstate: CPU model not supported
[    1.855587] hid: raw HID events driver (C) Jiri Kosina
[    1.857610] usbcore: registered new interface driver usbhid
[    1.859583] usbhid: USB HID core driver
[    1.861168] drop_monitor: Initializing network drop monitor service
[    1.863236] Initializing XFRM netlink socket
[    1.864847] NET: Registered protocol family 10
[    1.872823] Segment Routing with IPv6
[    1.874478] mip6: Mobile IPv6
[    1.875771] NET: Registered protocol family 17
[    1.877727] RAS: Correctable Errors collector initialized.
[    1.879999] IPI shorthand broadcast: enabled
[    1.881972] SSE version of gcm_enc/dec engaged.
[    1.934803] sched_clock: Marking stable (1638619910, 295496948)->(2257082768, -322965910)
[    1.938215] registered taskstats version 1
[    1.939988] Loading compiled-in X.509 certificates
[    1.995924] Loaded X.509 cert 'Fedora kernel signing key: 0d97d514d1651893ca89cc557d3830fffcdfaba2'
[    1.999485] zswap: loaded using pool lzo/zbud
[    2.001773] Key type ._fscrypt registered
[    2.003759] Key type .fscrypt registered
[    2.005676] Key type fscrypt-provisioning registered
[    2.018887] Key type big_key registered
[    2.025806] Key type encrypted registered
[    2.027703] ima: No TPM chip found, activating TPM-bypass!
[    2.030312] ima: Allocated hash algorithm: sha256
[    2.032602] ima: No architecture policies found
[    2.035485] PM:   Magic number: 8:220:121
[    2.037407] rtc_cmos 00:00: setting system clock to 2020-06-16T09:08:56 UTC (1592298536)
[    2.119404] ata3: SATA link down (SStatus 0 SControl 300)
[    2.121763] ata1: SATA link down (SStatus 0 SControl 300)
[    2.123965] ata4: SATA link down (SStatus 0 SControl 300)
[    2.126034] ata2: SATA link down (SStatus 0 SControl 300)
[    2.128247] ata5: SATA link down (SStatus 0 SControl 300)
[    2.130360] ata6: SATA link down (SStatus 0 SControl 300)
[    2.136596] Freeing unused decrypted memory: 2040K
[    2.139797] Freeing unused kernel image (initmem) memory: 2452K
[    2.141742] Write protecting the kernel read-only data: 22528k
[    2.145761] Freeing unused kernel image (text/rodata gap) memory: 2044K
[    2.149563] Freeing unused kernel image (rodata/data gap) memory: 1276K
[    2.201441] x86/mm: Checked W+X mappings: passed, no W+X pages found.
[    2.203523] rodata_test: all tests were successful
[    2.205899] x86/mm: Checking user space page tables
[    2.249169] x86/mm: Checked W+X mappings: passed, no W+X pages found.
[    2.252138] Run /init as init process
[    2.269297] systemd[1]: systemd v245.4-1.fc32 running in system mode. (+PAM +AUDIT +SELINUX +IMA -APPARMOR +SMACK +SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +SECCOMP +BLKID +ELFUTILS +KMOD +IDN2 -IDN +PCRE2 default-hierarchy=unified)
[    2.279896] systemd[1]: Detected virtualization kvm.
[    2.282308] systemd[1]: Detected architecture x86-64.
[    2.284782] systemd[1]: Running in initial RAM disk.

Welcome to Fedora 32 (Cloud Edition) dracut-050-26.git20200316.fc32 (Initramfs)!

[    2.290244] systemd[1]: No hostname configured.
[    2.292690] systemd[1]: Set hostname to <localhost>.
[    2.295410] systemd[1]: Initializing machine ID from KVM UUID.
[    2.438910] systemd[1]: Started Dispatch Password Requests to Console Directory Watch.
[  OK  ] Started Dispatch Password …ts to Console Directory Watch.
[    2.446752] systemd[1]: Reached target Local Encrypted Volumes.
[  OK  ] Reached target Local Encrypted Volumes.
[    2.452804] systemd[1]: Reached target Local File Systems.
[  OK  ] Reached target Local File Systems.
[    2.457551] systemd[1]: Reached target Paths.
[  OK  ] Reached target Paths.
[    2.461073] systemd[1]: Reached target Slices.
[  OK  ] Reached target Slices.
[    2.464666] systemd[1]: Reached target Swap.
[  OK  ] Reached target Swap.
[    2.468164] systemd[1]: Reached target Timers.
[  OK  ] Reached target Timers.
[    2.471608] systemd[1]: Listening on Journal Audit Socket.
[  OK  ] Listening on Journal Audit Socket.
[    2.475333] systemd[1]: Listening on Journal Socket (/dev/log).
[  OK  ] Listening on Journal Socket (/dev/log).
[    2.479426] systemd[1]: Listening on Journal Socket.
[  OK  ] Listening on Journal Socket.
[    2.482895] systemd[1]: Listening on udev Control Socket.
[  OK  ] Listening on udev Control Socket.
[    2.486543] systemd[1]: Listening on udev Kernel Socket.
[  OK  ] Listening on udev Kernel Socket.
[    2.490513] systemd[1]: Reached target Sockets.
[  OK  ] Reached target Sockets.
[    2.496336] systemd[1]: Starting Create list of static device nodes for the current kernel...
         Starting Create list of st…odes for the current kernel...
[    2.505885] systemd[1]: Starting Journal Service...
         Starting Journal Service...
[    2.509693] systemd[1]: Condition check resulted in Load Kernel Modules being skipped.
[    2.519518] systemd[1]: Starting Apply Kernel Variables...
         Starting Apply Kernel Variables...
[    2.537396] systemd[1]: Starting Setup Virtual Console...
         Starting Setup Virtual Console...
[    2.555220] systemd[1]: Finished Create list of static device nodes for the current kernel.
[  OK  ] Finished Create list of st… nodes for the current kernel.
[    2.719961] systemd[1]: Finished Apply Kernel Variables.
[  OK  ] Finished Apply Kernel Variables.
[    2.728170] systemd[1]: Finished Setup Virtual Console.
[  OK  ] Finished Setup Virtual Console.
[    2.737088] systemd[1]: Condition check resulted in dracut ask for additional cmdline parameters being skipped.
[    2.752256] systemd[1]: Starting dracut cmdline hook...
         Starting dracut cmdline hook...
[    2.767311] systemd[1]: Starting Create Static Device Nodes in /dev...
         Starting Create Static Device Nodes in /dev...
[    2.800211] systemd[1]: Finished Create Static Device Nodes in /dev.
[  OK  ] Finished Create Static Device Nodes in /dev.
[    2.925285] systemd[1]: Finished dracut cmdline hook.
[  OK  ] Finished dracut cmdline hook.
[    2.933390] systemd[1]: Starting dracut pre-udev hook...
         Starting dracut pre-udev hook...
[    3.005211] systemd[1]: Finished dracut pre-udev hook.
[  OK  ] Finished dracut pre-udev hook.
[    3.013683] systemd[1]: Starting udev Kernel Device Manager...
         Starting udev Kernel Device Manager...
[    3.052156] pcieport 0000:00:02.5: pciehp: Failed to check link status
[    3.495275] systemd[1]: Started Journal Service.
[  OK  ] Started Journal Service.
[    3.499928] audit: type=1130 audit(1592298537.961:2): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=kernel msg='unit=systemd-journald comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
         Starting Create Volatile Files and Directories...
[  OK  ] Finished Create Volatile Files and Directories.
[    3.544556] audit: type=1130 audit(1592298538.006:3): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=kernel msg='unit=systemd-tmpfiles-setup comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
[  OK  ] Started udev Kernel Device Manager.
[    3.722268] audit: type=1130 audit(1592298538.183:4): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=kernel msg='unit=systemd-udevd comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
         Starting udev Coldplug all Devices...
[  OK  ] Finished udev Coldplug all Devices.
[  OK  [    3.888473] audit: type=1130 audit(1592298538.349:5): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=kernel msg='unit=systemd-udev-trigger comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
] Reached target System Initialization.
[  OK  ] Reached target Basic System.
         Starting dracut initqueue hook...
         Mounting Kernel Configuration File System...
[  OK  ] Mounted Kernel Configuration File System.
[    4.051289] virtio_blk virtio2: [vda] 8388608 512-byte logical blocks (4.29 GB/4.00 GiB)
[    4.137743]  vda: vda1
[    4.151141] virtio_blk virtio3: [vdb] 736 512-byte logical blocks (377 kB/368 KiB)
[  OK  ] Found device /dev/disk/by-…4-3bbb-40b2-a6ba-f377f0c90217.
[  OK  ] Reached target Initrd Root Device.
[  OK  ] Finished dracut initqueue hook.
[[    4.395134] audit: type=1130 audit(1592298538.856:6): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=kernel msg='unit=dracut-initqueue comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
  OK  ] Reached target Remote File Systems (Pre).
[  OK  ] Reached target Remote File Systems.
         Starting File System Check…3bbb-40b2-a6ba-f377f0c90217...
[  OK  ] Finished File System Check…4-3bbb-40b2-a6ba-f377f0c90217.
[    4.519142] audit: type=1130 audit(1592298538.981:7): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=kernel msg='unit=systemd-fsck-root comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
         Mounting /sysroot...
[    4.545932] EXT4-fs (vda1): mounted filesystem with ordered data mode. Opts: (null)
[  OK  ] Mounted /sysroot.
[  OK  ] Reached target Initrd Root File System.
         Starting Reload Configuration from the Real Root...
[    4.584825] audit: type=1334 audit(1592298539.046:8): prog-id=7 op=UNLOAD
[    4.588199] audit: type=1334 audit(1592298539.050:9): prog-id=6 op=UNLOAD
[    4.595474] audit: type=1334 audit(1592298539.055:10): prog-id=5 op=UNLOAD
[  OK  ] Finished Reload Configuration from the Real Root.
[  OK  ] Reached target Initrd File Systems.
[  OK  ] Reached target Initrd Default Target.
         Starting dracut mount hook...
[  OK  ] Finished dracut mount hook.
         Starting Cleaning Up and Shutting Down Daemons...
[  OK  ] Finished Cleaning Up and Shutting Down Daemons.
[  OK  ] Stopped target Initrd Default Target.
[  OK  ] Stopped target Basic System.
[  OK  ] Stopped target Initrd Root Device.
[  OK  ] Stopped target Paths.
[  OK  ] Stopped target Remote File Systems.
[  OK  ] Stopped target Remote File Systems (Pre).
[  OK  ] Stopped target Slices.
[  OK  ] Stopped target Sockets.
[  OK  ] Stopped target System Initialization.
[  OK  ] Stopped target Local Encrypted Volumes.
[  OK  ] Stopped Dispatch Password …ts to Console Directory Watch.
[  OK  ] Stopped target Swap.
[  OK  ] Stopped target Timers.
[  OK  ] Stopped dracut mount hook.
[  OK  ] Stopped dracut initqueue hook.
[  OK  ] Stopped Apply Kernel Variables.
[  OK  ] Stopped Create Volatile Files and Directories.
[  OK  ] Stopped target Local File Systems.
[  OK  ] Stopped udev Coldplug all Devices.
         Stopping udev Kernel Device Manager...
[  OK  ] Stopped Setup Virtual Console.
[  OK  ] Stopped udev Kernel Device Manager.
[  OK  ] Closed udev Control Socket.
[  OK  ] Closed udev Kernel Socket.
[  OK  ] Stopped dracut pre-udev hook.
[  OK  ] Stopped dracut cmdline hook.
         Starting Cleanup udevd DB...
[  OK  ] Stopped Create Static Device Nodes in /dev.
[  OK  ] Stopped Create list of sta… nodes for the current kernel.
[  OK  ] Finished Cleanup udevd DB.
[  OK  ] Reached target Switch Root.
         Starting Switch Root...
[    5.128942] systemd-journald[291]: Received SIGTERM from PID 1 (systemd).
[    5.481780] kauditd_printk_skb: 30 callbacks suppressed
[    5.481782] audit: type=1404 audit(1592298539.943:41): enforcing=1 old_enforcing=0 auid=4294967295 ses=4294967295 enabled=1 old-enabled=1 lsm=selinux res=1
[    6.265603] SELinux:  Permission watch in class filesystem not defined in policy.
[    6.269820] SELinux:  Permission watch in class file not defined in policy.
[    6.272320] SELinux:  Permission watch_mount in class file not defined in policy.
[    6.275125] SELinux:  Permission watch_sb in class file not defined in policy.
[    6.277827] SELinux:  Permission watch_with_perm in class file not defined in policy.
[    6.281235] SELinux:  Permission watch_reads in class file not defined in policy.
[    6.284639] SELinux:  Permission watch in class dir not defined in policy.
[    6.288022] SELinux:  Permission watch_mount in class dir not defined in policy.
[    6.291767] SELinux:  Permission watch_sb in class dir not defined in policy.
[    6.295229] SELinux:  Permission watch_with_perm in class dir not defined in policy.
[    6.299139] SELinux:  Permission watch_reads in class dir not defined in policy.
[    6.302916] SELinux:  Permission watch in class lnk_file not defined in policy.
[    6.305417] SELinux:  Permission watch_mount in class lnk_file not defined in policy.
[    6.308010] SELinux:  Permission watch_sb in class lnk_file not defined in policy.
[    6.310559] SELinux:  Permission watch_with_perm in class lnk_file not defined in policy.
[    6.313320] SELinux:  Permission watch_reads in class lnk_file not defined in policy.
[    6.315924] SELinux:  Permission watch in class chr_file not defined in policy.
[    6.318943] SELinux:  Permission watch_mount in class chr_file not defined in policy.
[    6.322700] SELinux:  Permission watch_sb in class chr_file not defined in policy.
[    6.325539] SELinux:  Permission watch_with_perm in class chr_file not defined in policy.
[    6.328883] SELinux:  Permission watch_reads in class chr_file not defined in policy.
[    6.332111] SELinux:  Permission watch in class blk_file not defined in policy.
[    6.335612] SELinux:  Permission watch_mount in class blk_file not defined in policy.
[    6.339129] SELinux:  Permission watch_sb in class blk_file not defined in policy.
[    6.342557] SELinux:  Permission watch_with_perm in class blk_file not defined in policy.
[    6.346836] SELinux:  Permission watch_reads in class blk_file not defined in policy.
[    6.350886] SELinux:  Permission watch in class sock_file not defined in policy.
[    6.354741] SELinux:  Permission watch_mount in class sock_file not defined in policy.
[    6.358811] SELinux:  Permission watch_sb in class sock_file not defined in policy.
[    6.362787] SELinux:  Permission watch_with_perm in class sock_file not defined in policy.
[    6.367038] SELinux:  Permission watch_reads in class sock_file not defined in policy.
[    6.370622] SELinux:  Permission watch in class fifo_file not defined in policy.
[    6.373129] SELinux:  Permission watch_mount in class fifo_file not defined in policy.
[    6.375831] SELinux:  Permission watch_sb in class fifo_file not defined in policy.
[    6.378396] SELinux:  Permission watch_with_perm in class fifo_file not defined in policy.
[    6.381129] SELinux:  Permission watch_reads in class fifo_file not defined in policy.
[    6.384046] SELinux:  Class perf_event not defined in policy.
[    6.386439] SELinux:  Class lockdown not defined in policy.
[    6.388791] SELinux: the above unknown classes and permissions will be allowed
[    6.391866] SELinux:  policy capability network_peer_controls=1
[    6.394132] SELinux:  policy capability open_perms=1
[    6.395957] SELinux:  policy capability extended_socket_class=1
[    6.398913] SELinux:  policy capability always_check_network=0
[    6.401832] SELinux:  policy capability cgroup_seclabel=1
[    6.404509] SELinux:  policy capability nnp_nosuid_transition=1
[    6.423078] audit: type=1403 audit(1592298540.884:42): auid=4294967295 ses=4294967295 lsm=selinux res=1
[    6.432136] systemd[1]: Successfully loaded SELinux policy in 951.663ms.
[    6.583607] systemd[1]: Relabelled /dev, /dev/shm, /run, /sys/fs/cgroup in 64.890ms.
[    6.596232] systemd[1]: systemd v245.4-1.fc32 running in system mode. (+PAM +AUDIT +SELINUX +IMA -APPARMOR +SMACK +SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +SECCOMP +BLKID +ELFUTILS +KMOD +IDN2 -IDN +PCRE2 default-hierarchy=unified)
[    6.607950] systemd[1]: Detected virtualization kvm.
[    6.610449] systemd[1]: Detected architecture x86-64.

Welcome to Fedora 32 (Cloud Edition)!

[    6.618467] systemd[1]: Set hostname to <localhost.localdomain>.
[    6.626321] systemd[1]: Initializing machine ID from KVM UUID.
[    6.630868] systemd[1]: Installed transient /etc/machine-id file.
[    6.640609] audit: type=1334 audit(1592298541.102:43): prog-id=13 op=LOAD
[    6.644216] audit: type=1334 audit(1592298541.106:44): prog-id=13 op=UNLOAD
[    6.649667] audit: type=1334 audit(1592298541.106:45): prog-id=14 op=LOAD
[    6.652758] audit: type=1334 audit(1592298541.106:46): prog-id=14 op=UNLOAD
[    7.118717] systemd[1]: /usr/lib/systemd/system/sssd.service:13: PIDFile= references a path below legacy directory /var/run/, updating /var/run/sssd.pid → /run/sssd.pid; please update the unit file accordingly.
[    7.256488] systemd[1]: /usr/lib/systemd/system/sssd-kcm.socket:7: ListenStream= references a path below legacy directory /var/run/, updating /var/run/.heim_org.h5l.kcm-socket → /run/.heim_org.h5l.kcm-socket; please update the unit file accordingly.
[    7.331048] audit: type=1334 audit(1592298541.792:47): prog-id=15 op=LOAD
[    7.335142] audit: type=1334 audit(1592298541.797:48): prog-id=16 op=LOAD
[    7.340902] systemd[1]: initrd-switch-root.service: Succeeded.
[    7.344411] audit: type=1334 audit(1592298541.797:49): prog-id=17 op=LOAD
[    7.348307] systemd[1]: Stopped Switch Root.
[    7.350673] audit: type=1131 audit(1592298541.802:50): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='unit=systemd-journald comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
[  OK  ] Stopped Switch Root.
[    7.364914] systemd[1]: systemd-journald.service: Scheduled restart job, restart counter is at 1.
[    7.369991] systemd[1]: Created slice system-getty.slice.
[  OK  ] Created slice system-getty.slice.
[    7.376253] systemd[1]: Created slice system-modprobe.slice.
[  OK  ] Created slice system-modprobe.slice.
[    7.382554] systemd[1]: Created slice system-serial\x2dgetty.slice.
[  OK  ] Created slice system-serial\x2dgetty.slice.
[    7.389493] systemd[1]: Created slice system-sshd\x2dkeygen.slice.
[  OK  ] Created slice system-sshd\x2dkeygen.slice.
[    7.396273] systemd[1]: Created slice User and Session Slice.
[  OK  ] Created slice User and Session Slice.
[    7.400893] systemd[1]: Started Dispatch Password Requests to Console Directory Watch.
[  OK  ] Started Dispatch Password …ts to Console Directory Watch.
[    7.407154] systemd[1]: Started Forward Password Requests to Wall Directory Watch.
[  OK  ] Started Forward Password R…uests to Wall Directory Watch.
[    7.413838] systemd[1]: Set up automount Arbitrary Executable File Formats File System Automount Point.
[  OK  ] Set up automount Arbitrary…s File System Automount Point.
[    7.421696] systemd[1]: Reached target Local Encrypted Volumes.
[  OK  ] Reached target Local Encrypted Volumes.
[    7.427307] systemd[1]: Stopped target Switch Root.
[  OK  ] Stopped target Switch Root.
[    7.432098] systemd[1]: Stopped target Initrd File Systems.
[  OK  ] Stopped target Initrd File Systems.
[    7.437458] systemd[1]: Stopped target Initrd Root File System.
[  OK  ] Stopped target Initrd Root File System.
[    7.441338] systemd[1]: Reached target Paths.
[  OK  ] Reached target Paths.
[    7.444954] systemd[1]: Reached target Remote File Systems.
[  OK  ] Reached target Remote File Systems.
[    7.449786] systemd[1]: Reached target Slices.
[  OK  ] Reached target Slices.
[    7.453426] systemd[1]: Reached target Swap.
[  OK  ] Reached target Swap.
[    7.463225] systemd[1]: Listening on Process Core Dump Socket.
[  OK  ] Listening on Process Core Dump Socket.
[    7.467605] systemd[1]: Listening on initctl Compatibility Named Pipe.
[  OK  ] Listening on initctl Compatibility Named Pipe.
[    7.473108] systemd[1]: Listening on udev Control Socket.
[  OK  ] Listening on udev Control Socket.
[    7.477567] systemd[1]: Listening on udev Kernel Socket.
[  OK  ] Listening on udev Kernel Socket.
[    7.482069] systemd[1]: Listening on User Database Manager Socket.
[  OK  ] Listening on User Database Manager Socket.
[    7.487900] systemd[1]: Mounting Huge Pages File System...
         Mounting Huge Pages File System...
[    7.494354] systemd[1]: Mounting POSIX Message Queue File System...
         Mounting POSIX Message Queue File System...
[    7.506271] systemd[1]: Mounting Kernel Debug File System...
         Mounting Kernel Debug File System...
[    7.515226] systemd[1]: Mounting Kernel Trace File System...
         Mounting Kernel Trace File System...
[    7.523299] systemd[1]: Starting Create list of static device nodes for the current kernel...
         Starting Create list of st…odes for the current kernel...
[    7.533514] systemd[1]: Starting Load Kernel Module drm...
         Starting Load Kernel Module drm...
[    7.545913] systemd[1]: Condition check resulted in Set Up Additional Binary Formats being skipped.
[    7.554267] systemd[1]: Stopped Journal Service.
[  OK  ] Stopped Journal Service.
[    7.566383] systemd[1]: Starting Journal Service...
         Starting Journal Service...
[    7.578280] systemd[1]: Condition check resulted in Load Kernel Modules being skipped.
[    7.587337] systemd[1]: Condition check resulted in FUSE Control File System being skipped.
[    7.600421] systemd[1]: Starting Remount Root and Kernel File Systems...
         Starting Remount Root and Kernel File Systems...
[    7.620915] systemd[1]: Starting Repartition Root Disk...
         Starting Repartition Root Disk...
[    7.646316] systemd[1]: Starting Apply Kernel Variables...
         Starting Apply Kernel Variables...
[    7.670140] EXT4-fs (vda1): re-mounted. Opts: (null)
[    7.672374] systemd[1]: Starting udev Coldplug all Devices...
         Starting udev Coldplug all Devices...
[    7.702363] systemd[1]: sysroot.mount: Succeeded.
[    7.713501] systemd[1]: Mounted Huge Pages File System.
[  OK  ] Mounted Huge Pages File System.
[    7.731015] systemd[1]: Mounted POSIX Message Queue File System.
[  OK  ] Mounted POSIX Message Queue File System.
[    7.743616] systemd[1]: Mounted Kernel Debug File System.
[  OK  ] Mounted Kernel Debug File System.
[    7.752481] systemd[1]: Mounted Kernel Trace File System.
[  OK  ] Mounted Kernel Trace File System.
[    7.763267] systemd[1]: Finished Create list of static device nodes for the current kernel.
[  OK  ] Finished Create list of st… nodes for the current kernel.
[    7.776098] systemd[1]: modprobe@drm.service: Succeeded.
[    7.784403] systemd[1]: Finished Load Kernel Module drm.
[  OK  ] Finished Load Kernel Module drm.
[    7.795190] systemd[1]: Finished Remount Root and Kernel File Systems.
[  OK  ] Finished Remount Root and Kernel File Systems.
[    7.809400] systemd[1]: Finished Repartition Root Disk.
[  OK  ] Finished Repartition Root Disk.
[    7.821288] systemd[1]: Finished Apply Kernel Variables.
[  OK  ] Finished Apply Kernel Variables.
[    7.830684] systemd[1]: Condition check resulted in First Boot Wizard being skipped.
[    7.849017] systemd[1]: Starting Rebuild Hardware Database...
         Starting Rebuild Hardware Database...
[    7.865340] systemd[1]: Starting Load/Save Random Seed...
         Starting Load/Save Random Seed...
[    7.886352] systemd[1]: Starting Create System Users...
         Starting Create System Users...
[    7.947487] systemd[1]: Finished Load/Save Random Seed.
[  OK  ] Finished Load/Save Random Seed.
[    7.966085] systemd[1]: Finished Create System Users.
[  OK  ] Finished Create System Users.
[    7.974391] systemd[1]: Starting Create Static Device Nodes in /dev...
         Starting Create Static Device Nodes in /dev...
[    8.079271] systemd[1]: Finished Create Static Device Nodes in /dev.
[  OK  ] Finished Create Static Device Nodes in /dev.
[    8.100812] systemd[1]: Reached target Local File Systems (Pre).
[  OK  ] Reached target Local File Systems (Pre).
[    8.119098] systemd[1]: Reached target Local File Systems.
[  OK  ] Reached target Local File Systems.
[    8.144456] systemd[1]: Starting Restore /run/initramfs on shutdown...
         Starting Restore /run/initramfs on shutdown...
[    8.167430] systemd[1]: Condition check resulted in Import network configuration from initramfs being skipped.
[    8.189636] systemd[1]: Starting Rebuild Dynamic Linker Cache...
         Starting Rebuild Dynamic Linker Cache...
[    8.204125] systemd[1]: Condition check resulted in Mark the need to relabel after reboot being skipped.
[    8.218544] systemd[1]: Condition check resulted in Store a System Token in an EFI Variable being skipped.
[    8.252578] systemd[1]: Starting Commit a transient machine-id on disk...
         Starting Commit a transient machine-id on disk...
[    8.280441] systemd[1]: Finished udev Coldplug all Devices.
[  OK  ] Finished udev Coldplug all Devices.
[    8.300305] systemd[1]: Finished Restore /run/initramfs on shutdown.
[  OK  ] Finished Restore /run/initramfs on shutdown.
[    8.344249] systemd[1]: Finished Rebuild Dynamic Linker Cache.
[  OK  ] Finished Rebuild Dynamic Linker Cache.
[    8.361851] systemd[1]: etc-machine\x2did.mount: Succeeded.
[    8.367722] systemd[1]: Finished Commit a transient machine-id on disk.
[  OK  ] Finished Commit a transient machine-id on disk.
[    8.882928] systemd[1]: Started Journal Service.
[  OK  ] Started Journal Service.
         Starting Flush Journal to Persistent Storage...
[    8.920564] systemd-journald[501]: Received client request to flush runtime journal.
[  OK  ] Finished Flush Journal to Persistent Storage.
         Starting Create Volatile Files and Directories...
[  OK  ] Finished Create Volatile Files and Directories.
         Starting Security Auditing Service...
         Starting Rebuild Journal Catalog...
[  OK  ] Finished Rebuild Journal Catalog.
[  OK  ] Started Security Auditing Service.
         Starting Update UTMP about System Boot/Shutdown...
[  OK  ] Finished Update UTMP about System Boot/Shutdown.
[  OK  ] Finished Rebuild Hardware Database.
         Starting udev Kernel Device Manager...
         Starting Update is Completed...
[  OK  ] Finished Update is Completed.
[  OK  ] Started udev Kernel Device Manager.
[  OK  ] Reached target System Initialization.
[  OK  ] Started dnf makecache --timer.
[  OK  ] Started Discard unused blocks once a week.
[  OK  ] Started Daily Cleanup of Temporary Directories.
[  OK  ] Started daily update of the root trust anchor for DNSSEC.
[  OK  ] Reached target Timers.
[  OK  ] Listening on D-Bus System Message Bus Socket.
[  OK  ] Listening on SSSD Kerberos…ache Manager responder socket.
[  OK  ] Reached target Sockets.
[  OK  ] Reached target Basic System.
         Starting NTP client/server...
         Starting Initial cloud-init job (pre-networking)...
         Starting OpenSSH ecdsa Server Key Generation...
         Starting OpenSSH ed25519 Server Key Generation...
         Starting OpenSSH rsa Server Key Generation...
         Starting System Security Services Daemon...
         Starting Home Area Manager...
[  OK  ] Started NTP client/server.
[  OK  ] Finished OpenSSH ed25519 Server Key Generation.
[  OK  ] Finished OpenSSH ecdsa Server Key Generation.
[   12.153656] i801_smbus 0000:00:1f.3: SMBus using PCI interrupt
[  OK  ] Started System Security Services Daemon.
[  OK  ] Reached target User and Group Name Lookups.
         Starting Login Service...
         Starting D-Bus System Message Bus...
[  OK  ] Started D-Bus System Message Bus.
[   13.613714] bochs-drm 0000:00:01.0: vgaarb: deactivate vga console
[  OK  ] Started Home Area Manager.
[   13.690166] Console: switching to colour dummy device 80x25
[   13.750427] [drm] Found bochs VGA, ID 0xb0c0.
[   13.752159] [drm] Framebuffer size 16384 kB @ 0xfc000000, mmio @ 0xfea10000.
[   13.806157] [TTM] Zone  kernel: Available graphics memory: 106086 KiB
[   13.808780] [TTM] Initializing pool allocator
[   13.821141] [TTM] Initializing DMA pool allocator
[   13.832789] [drm] Found EDID data blob.
[   13.841950] [drm] Initialized bochs-drm 1.0.0 20130925 for 0000:00:01.0 on minor 0
[   13.869289] fbcon: bochs-drmdrmfb (fb0) is primary device
[   13.890181] Console: switching to colour frame buffer device 128x48
[   13.942758] bochs-drm 0000:00:01.0: fb0: bochs-drmdrmfb frame buffer device
[   13.980077] kvm: VM_EXIT_LOAD_IA32_PERF_GLOBAL_CTRL does not work properly. Using workaround
[  OK  ] Started Login Service.
[  OK  ] Finished OpenSSH rsa Server Key Generation.
[  OK  ] Reached target sshd-keygen.target.
         Starting Hostname Service...
[  OK  ] Started Hostname Service.
[   15.614254] cloud-init[629]: Cloud-init v. 19.4 running 'init-local' at Tue, 16 Jun 2020 09:09:09 +0000. Up 14.36 seconds.
[  OK  ] Finished Initial cloud-init job (pre-networking).
[  OK  ] Reached target Network (Pre).
         Starting Network Manager...
[  OK  ] Started Network Manager.
[  OK  ] Reached target Network.
         Starting Network Manager Wait Online...
         Starting Initial cloud-ini… (metadata service crawler)...
         Starting Network Manager Script Dispatcher Service...
[  OK  ] Started Network Manager Script Dispatcher Service.
[  OK  ] Finished Network Manager Wait Online.
[   18.479007] cloud-init[682]: Cloud-init v. 19.4 running 'init' at Tue, 16 Jun 2020 09:09:12 +0000. Up 18.13 seconds.
[   18.494936] cloud-init[682]: ci-info: +++++++++++++++++++++++++++++++++++Net device info++++++++++++++++++++++++++++++++++++
[   18.498840] cloud-init[682]: ci-info: +--------+------+-----------------------+---------------+--------+-------------------+
[   18.502930] cloud-init[682]: ci-info: | Device |  Up  |        Address        |      Mask     | Scope  |     Hw-Address    |
[   18.508027] cloud-init[682]: ci-info: +--------+------+-----------------------+---------------+--------+-------------------+
[   18.511584] cloud-init[682]: ci-info: |  eth0  | True |        10.0.2.2       | 255.255.255.0 | global | 02:00:00:73:41:92 |
[   18.517769] cloud-init[682]: ci-info: |  eth0  | True |    fd10:0:2::2/120    |       .       | global | 02:00:00:73:41:92 |
[   18.522490] cloud-init[682]: ci-info: |  eth0  | True | fe80::ff:fe73:4192/64 |       .       |  link  | 02:00:00:73:41:92 |
[   18.527110] cloud-init[682]: ci-info: |   lo   | True |       127.0.0.1       |   255.0.0.0   |  host  |         .         |
[   18.531740] cloud-init[682]: ci-info: |   lo   | True |        ::1/128        |       .       |  host  |         .         |
[   18.536718] cloud-init[682]: ci-info: +--------+------+-----------------------+---------------+--------+-------------------+
[   18.540856] cloud-init[682]: ci-info: +++++++++++++++++++++++++++Route IPv4 info++++++++++++++++++++++++++++
[   18.544946] cloud-init[682]: ci-info: +-------+-------------+----------+---------------+-----------+-------+
[   18.547787] cloud-init[682]: ci-info: | Route | Destination | Gateway  |    Genmask    | Interface | Flags |
[   18.550744] cloud-init[682]: ci-info: +-------+-------------+----------+---------------+-----------+-------+
[   18.554982] cloud-init[682]: ci-info: |   0   |   0.0.0.0   | 10.0.2.1 |    0.0.0.0    |    eth0   |   UG  |
[   18.557726] cloud-init[682]: ci-info: |   1   |   10.0.2.0  | 0.0.0.0  | 255.255.255.0 |    eth0   |   U   |
[   18.562373] cloud-init[682]: ci-info: +-------+-------------+----------+---------------+-----------+-------+
[   18.565214] cloud-init[682]: ci-info: ++++++++++++++++++++++Route IPv6 info+++++++++++++++++++++++
[   18.569717] cloud-init[682]: ci-info: +-------+----------------+-------------+-----------+-------+
[   18.573772] cloud-init[682]: ci-info: | Route |  Destination   |   Gateway   | Interface | Flags |
[   18.576477] cloud-init[682]: ci-info: +-------+----------------+-------------+-----------+-------+
[   18.580207] cloud-init[682]: ci-info: |   1   | fd10:0:2::/120 |      ::     |    eth0   |   U   |
[   18.583561] cloud-init[682]: ci-info: |   2   |   fe80::/64    |      ::     |    eth0   |   U   |
[   18.588508] cloud-init[682]: ci-info: |   3   |      ::/0      | fd10:0:2::1 |    eth0   |   UG  |
[   18.593434] cloud-init[682]: ci-info: |   5   |     local      |      ::     |    eth0   |   U   |
[   18.597769] cloud-init[682]: ci-info: |   6   |     local      |      ::     |    eth0   |   U   |
[   18.601345] cloud-init[682]: ci-info: |   7   |    ff00::/8    |      ::     |    eth0   |   U   |
[   18.605549] cloud-init[682]: ci-info: +-------+----------------+-------------+-----------+-------+
[   24.032459] cloud-init[682]: Generating public/private rsa key pair.
[   24.036277] cloud-init[682]: Your identification has been saved in /etc/ssh/ssh_host_rsa_key
[   24.039133] cloud-init[682]: Your public key has been saved in /etc/ssh/ssh_host_rsa_key.pub
[   24.042193] cloud-init[682]: The key fingerprint is:
[   24.044960] cloud-init[682]: SHA256:1l0jSI7JjPEJhWMIaTbS8yAZ91+Ti7YWvDbEwjfDLYo root@testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd
[   24.048511] cloud-init[682]: The key's randomart image is:
[   24.050390] cloud-init[682]: +---[RSA 3072]----+
[   24.053186] cloud-init[682]: |.+oo .oo. .      |
[   24.055697] cloud-init[682]: |+.O.. +B B .     |
[   24.058287] cloud-init[682]: | = =....@ o . o  |
[   24.060829] cloud-init[682]: |   ..= + + . o . |
[   24.062422] cloud-init[682]: |    o # S . .    |
[   24.065012] cloud-init[682]: |   . * O         |
[   24.066661] cloud-init[682]: |  E . *          |
[   24.069672] cloud-init[682]: |     o .         |
[   24.205897] cloud-init[682]: |                 |
[   24.208857] cloud-init[682]: +----[SHA256]-----+
[   24.211141] cloud-init[682]: Generating public/private dsa key pair.
[   24.214145] cloud-init[682]: Your identification has been saved in /etc/ssh/ssh_host_dsa_key
[   24.217720] cloud-init[682]: Your public key has been saved in /etc/ssh/ssh_host_dsa_key.pub
[   24.220684] cloud-init[682]: The key fingerprint is:
[   24.222851] cloud-init[682]: SHA256:ulGv76886y88wd1feUTUh+7E1CNf3Xgseu3PIPiFVKE root@testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd
[   24.226971] cloud-init[682]: The key's randomart image is:
[   24.228967] cloud-init[682]: +---[DSA 1024]----+
[   24.231691] cloud-init[682]: |              .B=|
[   24.233999] cloud-init[682]: |             o*.@|
[   24.236983] cloud-init[682]: |            E*o=+|
[   24.239048] cloud-init[682]: |            o =.o|
[   24.241728] cloud-init[682]: |        S. + * o.|
[   24.243842] cloud-init[682]: |       o .+ + =.+|
[   24.247374] cloud-init[682]: |      o  ..o o ++|
[   24.258284] cloud-init[682]: |       o o= .   +|
[   24.264860] cloud-init[682]: |      . .+BOo    |
[   24.267235] cloud-init[682]: +----[SHA256]-----+
[   24.269345] cloud-init[682]: Generating public/private ecdsa key pair.
[   24.272120] cloud-init[682]: Your identification has been saved in /etc/ssh/ssh_host_ecdsa_key
[   24.275272] cloud-init[682]: Your public key has been saved in /etc/ssh/ssh_host_ecdsa_key.pub
[   24.278896] cloud-init[682]: The key fingerprint is:
[   24.281465] cloud-init[682]: SHA256:LOaLvmWALdZ4d+Ws/Xuu/gCF4WHrSCiDQ2kQk5wSgBE root@testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd
[   24.287138] cloud-init[682]: The key's randomart image is:
[   24.289484] cloud-init[682]: +---[ECDSA 256]---+
[   24.291279] cloud-init[682]: |EBo.      +      |
[   24.293313] cloud-init[682]: |+=o.   . o =     |
[   24.297729] cloud-init[682]: |..o o . . = .    |
[   24.300342] cloud-init[682]: |   * o o * .     |
[   24.302641] cloud-init[682]: |  = = + S =      |
[   24.308820] cloud-init[682]: | . o = o o .     |
[   24.310802] cloud-init[682]: |      + . . .    |
[   24.313015] cloud-init[682]: |     + .   . ..  |
[   24.315201] cloud-init[682]: |   .+..    .=*o  |
[   24.317222] cloud-init[682]: +----[SHA256]-----+
[   24.319474] cloud-init[682]: Generating public/private ed25519 key pair.
[   24.322263] cloud-init[682]: Your identification has been saved in /etc/ssh/ssh_host_ed25519_key
[   24.325134] cloud-init[682]: Your public key has been saved in /etc/ssh/ssh_host_ed25519_key.pub
[   24.327694] cloud-init[682]: The key fingerprint is:
[   24.329784] cloud-init[682]: SHA256:WE+850o8+7k1tVfhsjNatJwOr5izEfjubWY/yYwWLoI root@testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd
[   24.334809] cloud-init[682]: The key's randomart image is:
[   24.337321] cloud-init[682]: +--[ED25519 256]--+
[   24.339973] cloud-init[682]: |                 |
[   24.341639] cloud-init[682]: |         .       |
[   24.343242] cloud-init[682]: |        . o    . |
[   24.345991] cloud-init[682]: |       o.o .  . .|
[   24.347843] cloud-init[682]: |      ..S.o .o .o|
[   24.349662] cloud-init[682]: |        ...+o =.o|
[   24.352836] cloud-init[682]: |      .  o=.*O+..|
[   24.356255] cloud-init[682]: |     E ..+=@*Bo..|
[   24.360199] cloud-init[682]: |        o*@=*=.  |
[   24.362457] cloud-init[682]: +----[SHA256]-----+
[  OK  ] Finished Initial cloud-ini…ob (metadata service crawler).
[  OK  ] Reached target Cloud-config availability.
[  OK  ] Reached target Network is Online.
         Starting Apply the settings specified in cloud-config...
         Starting OpenSSH server daemon...
         Starting Permit User Sessions...
[  OK  ] Started OpenSSH server daemon.
[  OK  ] Finished Permit User Sessions.
[  OK  ] Started Getty on tty1.
[  OK  ] Started Serial Getty on ttyS0.
[  OK  ] Reached target Login Prompts.
[  OK  ] Reached target Multi-User System.
         Starting Update UTMP about System Runlevel Changes...
[  OK  ] Finished Update UTMP about System Runlevel Changes.
         Stopping OpenSSH server daemon...
[  OK  ] Stopped OpenSSH server daemon.
[  OK  ] Stopped target sshd-keygen.target.
         Stopping sshd-keygen.target.
[  OK  ] Reached target sshd-keygen.target.
         Starting OpenSSH server daemon...
[  OK  ] Started OpenSSH server daemon.
[   27.210440] cloud-init[740]: Cloud-init v. 19.4 running 'modules:config' at Tue, 16 Jun 2020 09:09:21 +0000. Up 26.57 seconds.
[  OK  ] Finished Apply the settings specified in cloud-config.
         Starting Execute cloud user/final scripts...
[   28.914805] cloud-init[757]: Changing password for user fedora.
[   28.917905] cloud-init[757]: passwd: all authentication tokens updated successfully.
[   29.011489] cloud-init[757]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[   29.017415] cloud-init[757]:                                  Dload  Upload   Total   Spent    Left  Speed
  0  650k    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[   29.102048] cloud-init[757]: HTTP/1.1 200 OK
[   29.404338] cloud-init[757]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[   29.410511] cloud-init[757]:                                  Dload  Upload   Total   Spent    Left  Speed
100  650k  100  650k    0     0  24.4M      0 --:--:-- --:--:-- --:--:-- 24.4M
[   29.471940] cloud-init[757]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[   29.475939] cloud-init[757]:                                  Dload  Upload   Total   Spent    Left  Speed
100 24096  100 24096    0     0  1384k      0 --:--:-- --:--:-- --:--:-- 1470k
[  OK  ] Started /usr/local/bin/qemu-ga.
[   29.573821] cloud-init[757]: Running as unit: guestagent.service
ci-info: no authorized ssh keys fingerprints found for user fedora.
<14>Jun 16 09:09:24 ec2: 
<14>Jun 16 09:09:24 ec2: #############################################################
<14>Jun 16 09:09:24 ec2: -----BEGIN SSH HOST KEY FINGERPRINTS-----
<14>Jun 16 09:09:24 ec2: 1024 SHA256:ulGv76886y88wd1feUTUh+7E1CNf3Xgseu3PIPiFVKE root@testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd (DSA)
<14>Jun 16 09:09:24 ec2: 256 SHA256:LOaLvmWALdZ4d+Ws/Xuu/gCF4WHrSCiDQ2kQk5wSgBE root@testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd (ECDSA)
<14>Jun 16 09:09:24 ec2: 256 SHA256:WE+850o8+7k1tVfhsjNatJwOr5izEfjubWY/yYwWLoI root@testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd (ED25519)
<14>Jun 16 09:09:24 ec2: 3072 SHA256:1l0jSI7JjPEJhWMIaTbS8yAZ91+Ti7YWvDbEwjfDLYo root@testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd (RSA)
<14>Jun 16 09:09:24 ec2: -----END SSH HOST KEY FINGERPRINTS-----
<14>Jun 16 09:09:24 ec2: #############################################################
-----BEGIN SSH HOST KEY KEYS-----
ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBCUmy/Cbc5e5CWvEP1SyZCE1uI1FP8MfAUJlavpAA8D2Dr0ytQd33mmq0CAOmgAndvURJmnLWgVzmC+0M/X7Kvc= root@testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd
ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIBHbm0aW3MIAEyoljeiJTddVrDGJkGLcz6fHBo4MCTd/ root@testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDU99w8rDlhR8D7iacTu+herX2JMuBdXmherxuKUzwV8MuYIc9Xt2Y4XgtrLSrBiu0WHuSsljAnkuXJoNE/Z/FzVN4SQ7q/RsVbdgTWeCTjY+1iJImAtGAx5hOROZqDl28aSWwrgau4VVAtUhQmXRwT09nDU6E5aNvTIPwMeS8Z7kUE0I8tsgnE196ssxa8ZiQSL5focWbFw+x9KBtvwgFcMKBv3P3RpIBg/QrbJTg1M5iORX8qhbsUp2YuayGn1WY+DWE9KjDtzCSkHIkdDjgFz8CWyRZqFj+0jiuCmo1F1U59GzEgy9J2sMg6LNI4cQyC41RbREqv0nZmbttJVAErAG2go4zBAIGE9a+de6zllHskoT7LdzYv1S5uNaOvV9YEp5ADw+vRP3j6Jy9DvnhHeIz5E+vSe+6/EvAHD6ubDxqVszB+zMyU9CHJIDTNRG+uXyJL6wrjysbiSt+6KsY6/OxeoojMtPY/0egFDrUUYpLmdICa1MeS3Ynreqj5o/c= root@testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd
-----END SSH HOST KEY KEYS-----
[   29.850706] cloud-init[757]: Cloud-init v. 19.4 running 'modules:final' at Tue, 16 Jun 2020 09:09:23 +0000. Up 28.51 seconds.
[   29.854153] cloud-init[757]: ci-info: no authorized ssh keys fingerprints found for user fedora.
[   29.856682] cloud-init[757]: Cloud-init v. 19.4 finished at Tue, 16 Jun 2020 09:09:24 +0000. Datasource DataSourceNoCloud [seed=/dev/vdb][dsmode=net].  Up 29.82 seconds

Fedora 32 (Cloud Edition)
Kernel 5.6.6-300.fc32.x86_64 on an x86_64 (ttyS0)

SSH host key: SHA256:MWHRPU+4rTd8bb7agC+WI1fTpbQ0/ocPtvjECrTGGc4 (ECDSA)
SSH host key: SHA256:SPdzAFwkdqYrP5SurJzi5807eEXDA9Of/9xa07SMXJo (ED25519)
eth0: 10.0.2.2 fd10:0:2::2
testvmiplzw664p7ljt4dxwjd5rgk2txpxf29hhqkc422vmns8jlsgd login: 

```

Reply:
btw., the delay that it adds might affect behavior, just please consider
(worse case it will add flakiness, but on the other hand, we will make the code better at the end, matter of priority at the moment)

Reply:
What delay? This is always outputted but not printed... You claim that the print costs?
Also, if you claim that such a problem may occur, please provide numbers (how much time).

Reply:
I dont have numbers, i just say it might happen imo. and beside that, see the deterministic print above, just to consider if we do want it or not.

Reply:
Sorry, I do not understand. You added a comment and it is unclear what you suggest.
It seems you suggest that it should not be using this verbose flag because it adds delay and a long print.
If you have no numbers, I do not understand on what you base your delay addition.
And regarding the print, it is visible only when the test fails, so I do not understand the issue here.

Reply:
the print is visible always, on fedora (this what i remember from today run)
please try it locally

Reply:
As far as I know, only when a test fails stdout is presented (unless you setup it differently).
But you can see it easily on the CI jobs: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/3539/pull-kubevirt-e2e-k8s-1.17/1272744708010938368#1:build-log.txt%3A6788

Reply:
it appears in the stdout of the test when i run it locally, it means it does print it , hence consume time, will i wont argue for that.

Reply:
This writes all log messages to the GinkgoWriter which only writes to the terminal if a test fails: https://github.com/kubevirt/kubevirt/blob/master/tests/utils.go#L743. If a test fails we want the info.